src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -1147,6 +1173,145 @@ private static boolean shouldOmitClass(String classname) {
     }
   }
 
+  /**
+   * Returns classes in the given package. Does not include classes in sub-packages. Ignores
+   * non-accessible classes.
+   *
+   * @param packageName a package name
+   * @param accessibility the accessibility predicate
+   * @return classes in package {@code packageName}
+   */
+  public static Set<@ClassGetName String> getClassnamesFromPackage(
+      String packageName, AccessibilityPredicate accessibility) {
+    Set<@ClassGetName String> classnames = new HashSet<>();
+
+    for (File location : getClassLocationsForCurrentClasspath()) {
+      if (location.isFile() && location.getName().endsWith(".jar")) {
+        classnames.addAll(searchInJar(location, packageName, accessibility));
+      } else if (location.isDirectory()) {
+        classnames.addAll(searchInDirectory(location, packageName, accessibility));
+      }
+    }
+
+    return classnames;
+  }
+
+  /**
+   * Searches in the given directory for classes with the given package.
+   *
+   * @param directory a directory
+   * @param packageName a package name
+   * @param accessibility the accessibility predicate
+   * @return classes with the given package that were found in the specified directory
+   */
+  public static Set<@ClassGetName String> searchInDirectory(
+      File directory, String packageName, AccessibilityPredicate accessibility) {
+    String packageNameAsFile = packageName.replace(".", File.separator) /*.concat(File.separator)*/;
+    File packageDirectory =
+        new File(directory.getPath().concat(File.separator).concat(packageNameAsFile));
+    if (packageDirectory.exists()) {
+      try (Stream<Path> allFiles = Files.walk(Paths.get(packageDirectory.getPath()))) {
+        @SuppressWarnings("signature")
```

Comment:
Anytime you suppress a warning, write a comment to explain why the code is definitely correct and the suppressed warning is definitely a false positive warning.

Reply:
I needed to perform some operations to transform filename to ClassGetName form. Wrote comments explaining operations and why suppression is needed.